### PR TITLE
pre-alpha - [D-37449] - Fixing Meta information display in release, table column alignment and filter count display in filter components.

### PR DESCRIPTION
**Pull request by sishwarya**
## Release Checklist

- All steps below must be completed prior to merging.

## Changes Made
Fixed the following issue: 
1. The meta information display in the active release screen has been fixed.
   
Screenshot from 2024-10-25 09-32-47

2. Minor CSS alignment changes in the Template screen.
Screenshot from 2024-10-25 09-33-00
Screenshot from 2024-10-25 09-38-25

3. Display filter count in filter components.
Screenshot from 2024-10-25 11-32-20

### Required

- [ ] PR title should follow correct format.
  - MAJOR Release is used when there are breaking changes involved
  - MINOR Release is used when enhancements are introduced
  - PATCH Release is used for defects, security, tech chores, etc.
  - pre-alpha is used for testing releases
  - Note:
    - Based on the commit messages, increment the version from the latest release.
    - Push the bumped npm version in package.json back into the repo.
    - Push a tag for the new version back into the repo.
    - [For more details on versioning](https://github.com/phips28/gh-action-bump-version/blob/master/README.md) 
- [ ] Checklist of **changes made** added to PR description
- [ ] Related issue(s) linked to PR

 
## Reviewer Checklist

- [ ] Author checklist has been reviewed?
- [ ] All acceptance criteria has been met from linked issue
- [ ] Run code locally and verify all changes

## Related Issue(s)

Resolves:
Defect- D-37449 - Fix meta information displayed in an active release screen, Table header alignment in the Template screen and display filtercount in filter components.

### CODE REVIEW

**Review by Karthigaiselvan — APPROVED** [review]
LGTM

**Review by nitheshrayuduv — APPROVED** [review]
Looks good to me :)

File: plugins/dai-release/src/components/DenseScrollableTable/ScrollableTable.tsx [hunk 1]
```diff
@@ -110,12 +112,14 @@ export const ScrollableTable = ({
           <Table
             stickyHeader
             aria-label="sticky table"
-            style={{ tableLayout: 'auto' }}
```
Comment (nitheshrayuduv) [inline plugins/dai-release/src/components/DenseScrollableTable/ScrollableTable.tsx:113]: Looks Good to me :)

**Comment by nitheshrayuduv** [comment]
The New Changes with min Width and other fixes looks good.

style={{ tableLayout: 'fixed',minWidth: '75vw'}}

**Pull request merged**